nil/services/rpc/jsonrpc/types.go

```diff
@@ -93,6 +93,7 @@ type DebugRPCBlock struct {
 	OutTransactions []hexutil.Bytes        `json:"outTransactions"`
 	Receipts        []hexutil.Bytes        `json:"receipts"`
 	Errors          map[common.Hash]string `json:"errors"`
+	Config          map[string][]byte      `json:"config"`
```

Comment:
You need to return at least `map[string]hexutil.Bytes`. Or ideally you need to decode to some human-readable format.